docs/features/log-unhandled-exceptions.md

```diff
@@ -0,0 +1,36 @@
+# Arcus.WebApi.Logging.ExceptionHandlingMiddleware
```

Comment:
Can you add the Jekyll header please?

Think we can call this page `Logging`

Reply:
Like this ?